ddl/index.go

```diff
@@ -221,6 +235,23 @@ func (d *ddl) onCreateIndex(t *meta.Meta, job *model.Job) error {
 	}
 }
 
+func (d *ddl) convert2RollbackJob(t *meta.Meta, job *model.Job, tblInfo *model.TableInfo, indexInfo *model.IndexInfo) error {
+	job.State = model.JobRollback
+	job.Args = []interface{}{indexInfo.Name}
+	// If add index job rollbacks in write reorganization state, its need to delete all keys which has been added.
```

Comment:
It's OK. You can treat this matter with `drop index` job. The write reorganization state in add index job that likes write only state in drop index job.  WriteOnly(WriteReorg) -> DeleteOnly -> DeleteOnly -> DeleteReorg -> Absent.